src/mmw/js/src/modeling/models.js

```diff
@@ -276,10 +275,8 @@ var ScenarioModel = Backbone.Model.extend({
         this.set('inputs', new ModificationsCollection(attrs.inputs));
         this.set('modifications', new ModificationsCollection(attrs.modifications));
 
-        this.on('change:project change:name change:census', this.attemptSave, this);
-        this.get('inputs').on('add', this.attemptSave, this);
+        this.on('change:project change:name', this.attemptSave, this);
         this.get('modifications').on('add remove change', this.updateModificationHash, this);
-        this.get('modifications').on('add remove', this.attemptSave, this);
```

Comment:
This is true. I wonder if we can get @hectcastro to setup an alert / visualization to see average processing time for this to complete, and update our code based on that behavior. Also note that we poll for the job a limited number of times. If after that the job is still not complete, we abandon the results and save everything else.